tests/test_playbook.yml

```diff
@@ -7,6 +7,7 @@
       galaxy_tools_data_managers_list: files/run_data_managers.yaml
       galaxy_tools_galaxy_instance_url: http://localhost:8080/
       galaxy_tools_install_data_managers: yes
+      galaxy_config_style: "ini-paste"
```

Comment:
This is very useful. Would it be possible to add a task to differentiate between "ini-paste" and "yaml" by inspecting the extension of the `galaxy_config_file` variable or something similar, so that the user doesn't have to set an additional var?

Reply:
To be consistent with ansible-galaxy, what do you think about using the same [approach](https://github.com/galaxyproject/ansible-galaxy/blob/master/defaults/main.yml#L43), in which the style variable determines the value of the galaxy_config_style ?

Reply:
One major difference is that the ansible-galaxy playbook creates the config file, so it needs to know which format to create it in: https://github.com/galaxyproject/ansible-galaxy/blob/master/tasks/static_setup.yml#L26

galaxy-tools however, only modifies an existing config, so the format can be inferred from the file extension. I think this is worthwhile because it's one less variable to manually keep in sync?

Reply:
OK, I'll look at the reverse way.